custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -159,6 +164,32 @@ def updates(self):
             'start_date': self.month_start
         }
 
+        yield """
+        UPDATE "{tablename}" agg_awc SET
```

Comment:
Same comment as the other pr about adding more updates. This is going to be an incredibly slow step, if we do all of these bits as separate update queries